wallet/res/values/strings.xml

```diff
@@ -1,356 +1,396 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <resources>
 
```

Comment:
First, we don’t want whitespace changes.

Secondly, no Wall of Coins strings should be in this file.  They should be in strings-extra.xml to avoid conflicts with Bitcoin Wallet merges.